test/lib.test.js

```diff
@@ -68,12 +68,12 @@ const testVectorsSuccess = [
     transaction: {
       from: '0x2F764F3B669093dD24648757971070172Ca2af33',
       to: '0xA0FF2f1b0Ab2414E571bCD134781d746c750916B',
-      value: '50',
+      value: '44',
     },
     expected: {
-      maxFlowValue: '50',
```

Comment:
hummm... If I read the `'./test/graph-3.json'` it should be able to transfer 50... 🤔

Reply:
```{
  "maxFlowValue": "44",
  "transferSteps": [
    {
      "from": "0x2F764F3B669093dD24648757971070172Ca2af33",
      "step": 0,
      "to": "0xA0FF2f1b0Ab2414E571bCD134781d746c750916B",
      "token": "0x2F764F3B669093dD24648757971070172Ca2af33",
      "value": "30"
    },
    {
      "from": "0x2F764F3B669093dD24648757971070172Ca2af33",
      "step": 1,
      "to": "0xA0FF2f1b0Ab2414E571bCD134781d746c750916B",
      "token": "0xA0FF2f1b0Ab2414E571bCD134781d746c750916B",
      "value": "10"
    },
    {
      "from": "0x2F764F3B669093dD24648757971070172Ca2af33",
      "step": 2,
      "to": "0xA0FF2f1b0Ab2414E571bCD134781d746c750916B",
      "token": "0xc2f84dc24814999C7395cC0f553F1057Ee862a17",
      "value": "1"
    },
    {
      "from": "0x2F764F3B669093dD24648757971070172Ca2af33",
      "step": 3,
      "to": "0xA0FF2f1b0Ab2414E571bCD134781d746c750916B",
      "token": "0xE39608ddE072A2111936F4724eE927A20F597605",
      "value": "3"
    }
  ]
}
```